client/my-sites/sites/sites.jsx

```diff
@@ -27,6 +29,25 @@ export const Sites = React.createClass( {
 		path: React.PropTypes.string.isRequired
 	},
 
+	componentWillMount() {
```

Comment:
I think we can find better place for those redirects. For example in `client/my-sites/site-settings/index.js` there is a route that matches `/settings/:section` and calls `legacyRedirects`. Maybe that's the right place? The same should be valid for `/sites/:something`.

Reply:
Fine for settings, but what about sites? Can you be more specific - I'm not sure where to start looking for this.

Reply:
I believe this is the referenced method: [`legacyRedirects()`](https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/site-settings/controller.js#L159).

This might be better saved for a later refactor, as it would require a rewrite of the sites component to use a separate controller, etc. What do you think @hambai?